lib/bolt/transport/local/shell.rb

```diff
@@ -103,6 +103,10 @@ def check_sudo(out, inp, pid)
           # handle_sudo will return the line if it is not a sudo prompt or error
           lines.map! { |line| handle_sudo(inp, line, pid) }
           lines.join("")
+        # If err has reached EOF, no password prompt is expected
```

Comment:
This could be out too right?

Reply:
Yeah - I was thinking the sudo prompt only prints to err, but we run this check for both so it could be out that reached EOF. Good catch!